build/x86_64_groovy/Dockerfile

```diff
@@ -24,5 +24,8 @@ RUN apt-get -y update \
         make \
         pkg-config \
         python-dev-is-python2 \
+ && curl -sSO http://security.ubuntu.com/ubuntu/pool/main/c/coreutils/coreutils_8.30-3ubuntu2_amd64.deb \
```

Comment:
Since this package comes from Ubuntu Focal, you can literally just add the repo and install that specific version:
```
echo 'deb http://archive.ubuntu.com/ubuntu/ focal main' >/etc/apt/sources.list.d/focal.list
apt-get update
apt-get install -y --allow-downgrades coreutils=8.30-3ubuntu2
```
which is arguably cleaner than downloading the `.deb` file and installing locally.

Reply:
Tested above change which works

Hit:1 http://security.ubuntu.com/ubuntu groovy-security InRelease
Hit:2 http://archive.ubuntu.com/ubuntu groovy InRelease
Hit:3 http://archive.ubuntu.com/ubuntu groovy-updates InRelease
Hit:4 http://archive.ubuntu.com/ubuntu groovy-backports InRelease
Get:5 http://archive.ubuntu.com/ubuntu focal InRelease [265 kB]
Get:6 http://archive.ubuntu.com/ubuntu focal/main amd64 Packages [1275 kB]
Fetched 1540 kB in 1s (1256 kB/s)
Reading package lists...
Reading package lists...
Building dependency tree...
Reading state information...
The following packages will be DOWNGRADED:
  coreutils
0 upgraded, 0 newly installed, 1 downgraded, 0 to remove and 0 not upgraded.
Need to get 1249 kB of archives.
After this operation, 1077 kB disk space will be freed.
Get:1 http://archive.ubuntu.com/ubuntu focal/main amd64 coreutils amd64 8.30-3ubuntu2 [1249 kB]
debconf: delaying package configuration, since apt-utils is not installed
Fetched 1249 kB in 1s (1995 kB/s)
dpkg: warning: downgrading coreutils from 8.32-3ubuntu1 to 8.30-3ubuntu2
(Reading database ... 28476 files and directories currently installed.)
Preparing to unpack .../coreutils_8.30-3ubuntu2_amd64.deb ...
Unpacking coreutils (8.30-3ubuntu2) over (8.32-3ubuntu1) ...
Setting up coreutils (8.30-3ubuntu2) ...